ResidentContactApi/V1/Boundary/Response/InsertExternalResidentResponse.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ResidentContactApi.V1.Boundary.Response
+{
+    public class InsertExternalResidentResponse
```

Comment:
Could this be`InsertExternalReferenceRecordResponse` or similar rather than `InsertExternalResidentResponse`?